src/app/search/search-filters.ts

```diff
@@ -1018,11 +1018,20 @@ function searchFilters(
         );
       },
       modslot(item: DimItem, predicate: string) {
-        const modSocketHashes = seasonalSocketHashesByName[predicate];
+        if (!item.isDestiny2() || !item.sockets || !(item.bucket?.sort === 'Armor')) return false;
```

Comment:
```suggestion
        if (!item.isDestiny2() || !item.sockets || !(item.bucket?.sort === 'Armor')) {
          return false;
        }
```

Reply:
Sorry :( fixed.
My work environment auto runs formatters and linters when you create changes, so I have some bad habits :shame-cube:.